code/game/objects/items/weapons/material/thrown.dm

```diff
@@ -16,15 +17,12 @@
 
 /obj/item/weapon/material/star/throw_impact(atom/hit_atom)
 	..()
-	if(material.radioactivity>0 && istype(hit_atom,/mob/living))
+	if (istype(hit_atom,/mob/living))
+
 		var/mob/living/M = hit_atom
-		M.adjustToxLoss(rand(25,50))
 
-/obj/item/weapon/material/star/ninja
-	default_material = "uranium"
+		if(material.radioactivity > 0)
+			M.adjustToxLoss(material.radioactivity*2)
 
-/obj/item/weapon/material/star/ninja/throw_impact(atom/hit_atom)
-	..()
-	if(prob(30) && istype(hit_atom,/mob/living))
-		var/mob/living/M = hit_atom
-		M.Weaken(7)
\ No newline at end of file
+		if(prob(30))
+			M.Weaken(4)
```

Comment:
Weaken(4) vs Weaken(7)

Reply:
I feel that 7 is too strong. 4 seems fair.

Reply:
I can change it back to 7 if you wish.

Reply:
Hm, lets keep it at 7 for now and see if there are any issues.